scripts/release/stage-release.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -ex
+
+# Stages a release by putting everything that should be packaged and released
+# into the ./deploy folder. This script should be run from the root of the
+# material2 repo.
+
+# Make sure you are not running `ng serve` or `ng build --watch` when running this.
+
+
+# Clear dist/ and deploy/ so that we guarantee there are no stale artifacts.
+rm -rf ./dist
+rm -rf ./deploy
+
+# For packaging for npm only, replace the TypeScript module format to commonjs
```

Comment:
it's not obvious for the casual reader that this will create tsconfig.json.backup, maybe state that explicitly in the comment